src/Controls/src/Core/Compatibility/Android/PlatformInvalidate.cs

```diff
@@ -8,14 +8,13 @@ class PlatformInvalidate : IPlatformInvalidate
 	{
 		public void Invalidate(VisualElement visualElement)
 		{
-			var renderer = visualElement.GetRenderer();
-			if (renderer == null || renderer.View.IsDisposed())
-			{
+			if (visualElement.Handler?.PlatformView == null)
 				return;
-			}
 
-			renderer.View.Invalidate();
-			renderer.View.RequestLayout();
+			var view = visualElement.ToPlatform();
```

Comment:
What happens for views that only have a renderer? Or is everything now a handler and then maybe we don't need this at all because we just use it in one place:

https://github.com/dotnet/maui/blob/8dd650231fc535f0475699bb2c7eacd3cbc30461/src/Controls/src/Core/VisualElement.cs?q=IPlatformInvalidate#L726

The only reason it is a service is because it _was_ a renderer in the compat dll.

Reply:
> What happens for views that only have a renderer? Or is everything now a handler and then maybe we don't need this at all because we just use it in one place:

Everything always received a `Handler` even if it's using a shim the `Handler` property would still get set. 

> The only reason it is a service is because it was a renderer in the compat dll.

Yea, I'm trying to limit the amount of opinion this PR takes about things. There's a handful of places in controls (RadioButton, SwipeView, etc..) that still use these things. So, we need to fix all of that. I didn't want to fix all of that here. More PRs incoming where those changes can be better discussed and seen